cypress/cypress/e2e/edit-lesson.cy.ts

```diff
@@ -130,57 +130,55 @@ describe("edit lesson screen", () => {
         gqlQueries: [mockGQL("FetchLesson", { me: { lesson } })],
       });
       cy.visit("/lessons/edit");
-      cy.get("[data-cy=lesson-name]").within(($input) => {
-        cy.get("textarea").should("have.value", "Display name for the lesson");
-      });
+      // cy.get("[data-cy=lesson-name]").within(($input) => {
+      //   cy.get("textarea").should("have.value", "Display name for the lesson");
+      // });
       cy.get("[data-cy=lesson-creator]").within(($input) => {
         cy.get("textarea").should("have.value", "Kayla");
       });
-      cy.get("[data-cy=intro]").within(($input) => {
-        cy.get("textarea").should(
-          "have.value",
-          "Introduction to the lesson,  e.g. 'This is a lesson about RGB colors'"
-        );
-      });
-      cy.get("[data-cy=question]").within(($input) => {
-        cy.get("textarea").should(
-          "have.value",
-          "Question the student needs to answer, e.g. 'What are the colors in RGB?'"
-        );
-      });
+      // cy.get("[data-cy=intro]").within(($input) => {
+      //   cy.get("textarea").should(
+      //     "have.value",
+      //     "Introduction to the lesson,  e.g. 'This is a lesson about RGB colors'"
+      //   );
+      // });
+      // cy.get("[data-cy=question]").within(($input) => {
+      //   cy.get("textarea").should(
+      //     "have.value",
+      //     "Question the student needs to answer, e.g. 'What are the colors in RGB?'"
+      //   );
+      // });
       cy.get("[data-cy=classifier-arch]").contains("LR2");
       cy.get("[data-cy=lesson-format]").contains("Default");
-      cy.get("[data-cy=media-type]").contains("None");
+      cy.get("[data-cy=video-url]").should("not.exist");
+      cy.get("[data-cy=image-url]").should("not.exist");
```

Comment:
should have some way to test for images and video, but that may not be easy with the framework.